spec/api-session-spec.js

```diff
@@ -289,12 +289,16 @@ describe('session module', () => {
       res.end(mockPDF)
       downloadServer.close()
     })
+
+    const isPathEqual = (path1, path2) => {
+      return path.relative(path1, path2) === ''
```

Comment:
I know it's kinda an edge case but this will glitch out if `path.relative` receives a non-absolute path.  `path` will automatically try resolve it based on the CWD.

Can we just `toLowerCase()` to solve your issue?

Reply:
Yes, we can. I have just hoped there is some system solution which can compare paths. For example on case sensitive filesystem it would really mean different path, so the test should fail. Case sensitive FS is mostly used on iOS and Linux so 'toLowerCase()' will return false positives on these systems.
I will try to resolve relative paths. I I am unable to do so, I will use toLowerCase()...

Reply:
@torycl Maybe something like

```js
const isPathEqual = (p1, p2) => {
  const pp1 = path.parse(p1)
  const pp2 = path.parse(p2)
  return pp1.dir.substr(pp1.root.length) === pp2.dir.substr(pp2.root.length)
    && pp1.base === pp2.base;
}
```

Reply:
Would prefer to not use `toLowerCase()` on case-sensitive filesystems.

path.resolve() seems to be the only node.path function that does any case conversion on the windows root, so it looks like the shortest path to handling this gracefully & portably.

@MarshallOfSound could you expand on the edge case you're referring to? Is there a path to it occurring in how this spec uses `isPathEqual()`?

Reply:
It is super edge case but basically if one path returns "a" and the expected path happens to be "CWD/a" using path.relative will internally resolve those paths and declare them equal.  As mentioned it's an edge case but thought it was worth bringing up

Reply:
Ah, so it can depend on the position where the test has been launched. 
But if the parameter "savePath" is relative path the test will not fail if it is called from the right folder. Do we want to catch the situation when "savePath" is not absolute? Is it an error?

Reply:
I believe it depends on the API.  I can't think of any apis that should return relative paths.  I think any non-absolute path should fail the test

Reply:
Added check for any non-absolute paths. Can you merge please?